src/main/java/pl/nogacz/chess/board/Board.java

```diff
@@ -181,6 +197,166 @@ public void readMouseEvent(MouseEvent event) {
         }
     }
 
+    public void playerVsPlayer(MouseEvent event){
+        if(isWhitesTurn && !isBlacksTurn){
+            if(isBlacksTurn || isGameEnd) {
+                return;
+            }
+
+            Coordinates eventCoordinates = new Coordinates((int) ((event.getX() - 37) / 84), (int) ((event.getY() - 37) / 84));
+
+            if(!eventCoordinates.isValid()) {
+                return;
+            }
+
+            gameLogic.prepareData();
+
+            if(!gameLogic.isMovePossible()) {
+                noMovePossibleInfo();
+            }  else if(isKingChecked && possiblePawnIfKingIsChecked.size() == 0) {
+                if(gameLogic.isKingChecked(PawnColor.BLACK)) {
+                    statistics.addGameWin();
+                    endGame("White win! Congratulations :)");
+                } else {
+                    statistics.addGameLoss();
+                    endGame("White loss. Maybe you try again?");
+                }
+            } else {
+                if(isSelected) {

```

Comment:
what is isSelected?

Reply:
Yeniden adlandırıp isPieceSelected şeklinde güncelledim